queries.sql

```diff
@@ -24,4 +24,5 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
## [OPTIONAL] 
* This is a better way to write this query as it is easier to read `SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12'`

Reply:
Thank you for the helpful feedback